config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -118,7 +118,7 @@ presubmits:
       - release-0.2
     path_alias: sigs.k8s.io/cluster-api
     optional: true
-    always_run: true
+    run_if_changed: '\.go$|Makefile|\.sh$'
```

Comment:
Might be worth also to add `go.mod` files ?

Reply:
added!